controllers/publish-pane.js

```diff
@@ -23,18 +34,95 @@ module.exports = function () {
           progress.done('error');
           pane.openValidationErrors(errors);
         } else {
-          return edit.publishPage()
-            .then(function (url) {
-              progress.done();
-              progress.open('publish', `Published! <a href="${url}" target="_blank">View Article</a>`);
-            })
-            .catch(function () {
-              // note: the Error passed into this doesn't have a message, so we use a custom one
-              progress.done('error');
-              progress.open('error', `A server error occured. Please try again.`, true);
-            });
+          return edit.unschedulePublish(pageUri).then(function () {
+            return edit.publishPage()
+              .then(function (url) {
+                progress.done();
+                progress.open('publish', `Published! <a href="${url}" target="_blank">View Article</a>`);
+                state.toggleScheduled(false);
+              })
+              .catch(function () {
+                // note: the Error passed into this doesn't have a message, so we use a custom one
+                progress.done('error');
+                progress.open('error', `A server error occured. Please try again.`, true);
```

Comment:
Perhaps error messages should say what was trying to be accomplished?
"Failed to Unschedule. Server error. Please try again."